tools/unit_test_runner.sh

```diff
@@ -62,7 +62,7 @@ for pkg in $flaky_tests; do
   max_attempts=3
   attempt=1
   # Set a timeout because some tests may deadlock when they flake.
-  until go test -timeout 30s $VT_GO_PARALLEL $pkg -count=1; do
+  until go test -timeout 2m $VT_GO_PARALLEL $pkg -count=1; do
```

Comment:
why do we need a longer timeout?

Reply:
I saw a few cases where the VReplication test got killed after 30s (saw one today). On my laptop, it runs for 15-20 seconds. So, we have to give more time for these VMs that have unreliable CPU performance.

Most of this time is with mysql coming up and shutting down. If we speed that up, we can revert back to 30s.